Makefile

```diff
@@ -50,6 +51,13 @@ bin:
 	@mkdir -p $(BINDIR)
 	GOOS=linux $(GO) build -o $(BINDIR) $(GOFLAGS) -ldflags '$(LDFLAGS)' antrea.io/antrea/cmd/...
 
+trivy:
+	curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b $(BINDIR)
```

Comment:
usually we put installed tools in a different bin directory than antrea artifacts (see other examples in the makefile)